packages/ext-angular-tests/src/app/test/Lifecycle/MyComponent/MyComponent.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+
+declare var Ext: any;
+
+@Component({
+  selector: 'my-component',
+  templateUrl: './MyComponent.html',
+})
+
```

Comment:
Could you remove all new lines after the decorator and before the class? 

Here is a good example: https://angular.io/guide/architecture-components#component-metadata

Reply:
Fixed.